open_lisa/domain/filesystem/filemanager.py

```diff
@@ -31,11 +33,24 @@ def list_directory_recursively(self, directory):
             as_dict[base_key])
         return directory_as_list
 
+    @staticmethod
+    def save_command_result(file_path):
+        file_path = FileManager.get_file_path(file_path)
+        file_exists = os.path.exists(file_path)
+        if file_exists:
+            logging.info("[OpenLISA][ServerProtocol][handle_delete_file] Deleting file in {}".format(file_path))
+            os.remove(file_path)
+
     @staticmethod
     def get_file_path(file_name):
         while file_name.startswith('/') or file_name.startswith('\\') or file_name.startswith('.'):
             file_name = file_name[1:]
 
+        if not file_name.startswith(os.getenv("USER_FILES_FOLDER")) \
```

Comment:
En esta parte habría que partir la variable `file_path` en dos (suponiendo que tiene la siguiente estructura: `sandbox/path/specific/file.extension`
- root_folder: `sandbox`
- file_name: `file.extension`

y previamente validaría que `file_path` no contiene "." para evitar navegaciones hacia atrás en el filesystem

Reply:
Y una vez obtenido `root_folder` podemos hacer la validación `if root_folder not in SUPPORTED_FOLDERS: raise`